### ISSUE TITLE
Add personal configurations while provisioning

### ISSUE THREAD

**Issue by pulinagrawal** [CognitiveComputingResearchGroup/common-ccrgdev#3]
Hold on! :D 
This should not be against vagrant principles. 

What I mean is we can add some personal configurations like setting git name and email or anything else we may be able to find. 

Let me know if you feel that's ok @skugele

**Comment by skugele** [comment]
What did you have in mind?  Not clear to me how you plan on adding user specific settings to the provisioning file.  Details pleasssse.

**Comment by pulinagrawal** [comment]
I was imagining that when vagrant provisions a machine it could ask user for user.email and user.name flags that need to be set before using git. 

I really cannot think of any other customizations right now. 

This is definitely not at all important. Just some thoughts I had while I was provisioning my machine. 

Some problems I faced were installation of VBGuestAdditions and blank icons. 
I had to manually install vbguestadditions and change icon theme to get back icons.

**Comment by skugele** [comment]
I wonder if we should pull the box "customizations" from a file rather than prompting the user for it each time we provision a new vagrant box.  One of the thing that annoys me is having to start all over each time I create a new vagrant box.  We could take advantage of the fact that we have a shared directory between the host machine and the vm, so that we can place a customization file (or script) on the host, and then read those values into .bashrc or execute a set of commands whenever we provision the box.

Glad you mentioned the fix for the missing icons.  That has been annoying me for a while, but not enough to try to fix it.  :)  We should try to find a way to set the icon theme in the provisioning file.  I will create an issue tracker for it.

**Comment by pulinagrawal** [comment]
Absolutely. Using a file was the long route that I was hesitant to mention for just two things, namely email and username. But I am strongly in favor of that.
